weather_mv/loader_pipeline/util_test.py

```diff
@@ -178,5 +185,99 @@ def test_make_attrs_ee_compatible_a2(self):
 
 
 class ToJsonSerializableTypeTests(unittest.TestCase):
-    # TODO(#106): Write tests...
-    pass
+
+    def test_to_json_serializable_type_none(self):
+        attrs = None
+
+        expected = None
+
+        actual = to_json_serializable_type(attrs)
+
+        self.assertEqual(actual, expected)
```

Comment:
I know AAA is standard, but I think it would be more readable if we had a series of checks that all fit on one line that were mostly the same, where we only needed to check in the value being compared against. For example: 

```suggestion
    self.assertIsNone(self.convert(None))
    self.assertIsNone(self.convert(float('NaN')))
    self.assertIsNone(self.convert(np.nan))
    self.assertIsNotNone(self.convert([]))
```

Reply:
I agree, it looks much better and compressed when we have the checks inline. I went ahead and created a wrapper function `_convert` to wrap `to_json_serializable_type`.

Reply:
Added in https://github.com/google/weather-tools/pull/327/commits/d9cf97add436e462d01fb43fe2d32ac7c0511d17.